### PR TITLE
fix: TR-2486. Enhance design according to changes

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-2486

What to fixed: 
1. changed lable for review panel. (previously design and AC were misaligned)
2. Changed button name accrodingly
3. Changed a little behavior for button  during open/close panel

How to test: run test and check ACs

### CODE REVIEW

File: src/plugins/navigation/review/review.js [hunk 1]
```diff
@@ -447,6 +470,13 @@ export default pluginFactory({
         } else {
             this.flagItemButton.turnOff();
         }
+        if (this.customLayout) {
+            if (!this.explicitlyHidden) {
+                this.toggleButton.turnOn();
+            } else {
+                this.toggleButton.turnOff();
```
Comment (olga-kulish) [inline src/plugins/navigation/review/review.js:477]: Also add `this.toggleButton.turnOff();` to `disable()` method, for consistency with `flagItemButton`?

File: src/plugins/navigation/review/navigatorFizzy.tpl [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="qti-panel qti-navigator{{#if hidden}} hidden{{/if}} fizzy">
-    <span class="qti-navigator-label" title="{{__ 'Test review'}}">
-            <span class="qti-navigator-text">{{__ 'Test review'}}</span>
+    <span class="qti-navigator-label" title="{{__ 'Test overview'}}">
+            <span class="qti-navigator-text">{{__ 'Test overview'}}</span>
```
Comment (olga-kulish) [inline src/plugins/navigation/review/navigatorFizzy.tpl:3]: Double-check: "Test Overview" or "Test overview"?

**Review by olga-kulish — COMMENTED** [review]
Please update unit test, it's failing now

**Review by olga-kulish — COMMENTED** [review]
I also see QA's comment about linear test part mixed with non-linear parts. I assume for this PR it stays as before?

**Comment by svoronuk** [comment]
> I also see QA's comment about linear test part mixed with non-linear parts. I assume for this PR it stays as before?

Yeah. PM decided do not take into account this case.

File: src/plugins/navigation/review/review.js (hunk 1 above)
Comment (svoronuk) [reply src/plugins/navigation/review/review.js:477]: Absolutely right. Good catch.

File: src/plugins/navigation/review/navigatorFizzy.tpl (hunk 2 above)
Comment (svoronuk) [reply src/plugins/navigation/review/navigatorFizzy.tpl:3]: Test overview.

File: test/plugins/navigation/review/navigator/test.js [hunk 3]
```diff
@@ -488,7 +488,36 @@ define([
...
+                'Review panel title still exists'
+            );
+
+            ready();
+        })
+        .render($container)
+        .update(sample.map, sample.context);
+    });
+
+    QUnit.test('check open button flow', function(assert) {
```
Comment (olga-kulish) [inline test/plugins/navigation/review/navigator/test.js:501]: Isn't this test a copy of `QUnit.test('check disabling section titles', function(assert) {` but with undefined value for `displaySectionTitles`? So basically what you are checking is that default value for this option is `false`. If I'm right, maybe rename this test case to e.g."_check default for section titles option_"?

**Review by olga-kulish — APPROVED** [review]
Works nicely

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by oatymart — APPROVED** [review]
Changes seem ok to me, and tests are passing.

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Documentation is updated according to changes (if applicable)

File: test/plugins/navigation/review/navigator/test.js (hunk 3 above)
Comment (svoronuk) [reply test/plugins/navigation/review/navigator/test.js]: This was try to check close/open functionality. Unfortunatley QUnit do not have method like in Svelte `tick()`. I tried to find a way to wait tile panel was updated finally + find method where i can call `ready()`, but didn't find this. As of complexity of such investigation much more, than profit of the test, so i remove it. May be after finalizing my tasks i can return to this problem.

**Pull request merged**